tests/Checker/Checks_Tests.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Tests for the Checks class.
+ *
+ * @package plugin-check
+ */
+
+use Exception;
+use WordPress\Plugin_Check\Checker\Check_Result;
+use WordPress\Plugin_Check\Checker\Checks;
+
+class Checks_Tests extends WP_UnitTestCase {
+
+	protected $checks;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->checks = new Checks( 'test-plugin/test-plugin.php' );
+	}
+
+	public function test_get_checks_returns_array_of_expected_checks() {
```

Comment:
@jjgrainger Can we add some simple function doc here, so it will be easy to know which test function covers which actual function? something like below:

```
/**
 * @covers Checks::get_checks()
 */
 ```

Reply:
Thanks @vishalkakadiya the method being tested is actually covered by the test name `test_get_checks...` so there is no need for a docblock for this.